docs/DOCKER.rst

```diff
@@ -47,19 +39,28 @@ For running an S3 AWS backend, you will have to add a new section
             "credentialsProfile": "default"
         }
     }
+(...)
 
-You will also have to mount your AWS credentials file:
+You will also have to edit your AWS credentials file to be able to use your
+command line tool of choice. This file should only state credentials for S3 AWS
+hosted data, not for your Zenko CloudServer hosted data.
+As for your locationConfig.json, you will need to mount your AWS credentials
+file at run time:
 ``-v ~/.aws/credentials:/root/.aws/credentials`` on Linux, OS X, or Unix or
 ``-v C:\Users\USERNAME\.aws\credential:/root/.aws/credentials`` on Windows
 
-.. code:: shell
+S3DATA=file
+^^^^^^^^^^^
+When storing file data, for it to be persistent you must mount docker volumes
+for both data and metadata. See `this section <#using-docker-volumes-in-production>`__
 
-    docker run --name s3server -p 8000:8000
-    -v $(pwd)/locationConfig.json:/usr/src/app/locationConfig.json
-    -v ~/.aws/credentials:/root/.aws/credentials -e S3DATA=multiple scality/s3server
+S3DATA=mem
+^^^^^^^^^^
+This is ideal for testing - no data will remain after container is shutdown. Setting
+S3BACKEND=mem will achieve the same result.
```

Comment:
Please remove S3DATA=mem reference.

Reply:
Do you mean we should only mention S3BACKEND=mem? Or not mention the mem mode at all? From what I see at events, people use in-memory storage a lot for testing purposes.

Reply:
Changed